eng/helix_xharness.proj

```diff
@@ -0,0 +1,126 @@
+        <HelixPreCommand Include="export XHARNESS_LOG_WITH_TIMESTAMPS=true" />
+    </ItemGroup>
+
+    <ItemGroup>
+
+        <MAUIScenario Include="Controls">
+            <ScenarioDirectoryName>Controls.DeviceTests</ScenarioDirectoryName>
+            <PayloadDirectory>$(ScenariosDir)Controls.DeviceTests</PayloadDirectory>
+            <IPAName>Microsoft.Maui.Controls.DeviceTests</IPAName>
+            <PackageName>com.microsoft.maui.controls.devicetests</PackageName>
+        </MAUIScenario>
+
+       <MAUIScenario Include="Core">
+            <ScenarioDirectoryName>Core.DeviceTests</ScenarioDirectoryName>
+            <PayloadDirectory>$(ScenariosDir)Core.DeviceTests</PayloadDirectory>
+            <IPAName>Microsoft.Maui.Core.DeviceTests</IPAName>
+            <PackageName>com.microsoft.maui.core.devicetests</PackageName>
+        </MAUIScenario>
+
+        <MAUIScenario Include="Graphics">
+            <ScenarioDirectoryName>Graphics.DeviceTests</ScenarioDirectoryName>
+            <PayloadDirectory>$(ScenariosDir)Graphics.DeviceTests</PayloadDirectory>
+            <IPAName>Microsoft.Maui.Graphics.DeviceTests</IPAName>
+            <PackageName>com.microsoft.maui.graphics.devicetests</PackageName>
+        </MAUIScenario>
+
+        <MAUIScenario Include="Essentials">
+            <ScenarioDirectoryName>Essentials.DeviceTests</ScenarioDirectoryName>
+            <PayloadDirectory>$(ScenariosDir)Essentials.DeviceTests</PayloadDirectory>
+            <IPAName>Microsoft.Maui.Essentials.DeviceTests</IPAName>
+            <PackageName>com.microsoft.maui.essentials.devicetests</PackageName>
+        </MAUIScenario>
+
+        <MAUIScenario Include="MauiBlazorWebView">
+            <ScenarioDirectoryName>MauiBlazorWebView.DeviceTests</ScenarioDirectoryName>
+            <PayloadDirectory>$(ScenariosDir)MauiBlazorWebView.DeviceTests</PayloadDirectory>
+            <IPAName>Microsoft.Maui.MauiBlazorWebView.DeviceTests</IPAName>
+            <PackageName>com.microsoft.maui.mauiblazorwebview.devicetests</PackageName>
+        </MAUIScenario>
+    </ItemGroup>
+
+    <!-- Process each test scenario individually using direct path construction -->
+    <Target Name="DiscoverTestBundles" BeforeTargets="Build">
+        <!-- Print all MAUIScenarios -->
+        <Message Text="Discovered MAUIScenarios: @(MAUIScenario->Count())" Importance="high" />
+
+        <ItemGroup>
+            <!-- Create a special batching task to find app bundles by scenario -->
+            <_MAUIScenarioSearch Include="@(MAUIScenario)" />
+        </ItemGroup>
+
+        <!-- Add the discovered items to the appropriate Helix test collection -->
+        <ItemGroup Condition="'$(TargetOS)' == 'ios'">
+            <XHarnessAppBundleToTest Include="$([System.IO.Directory]::GetDirectories('%(_MAUIScenarioSearch.PayloadDirectory)/Release/net9.0-ios/', '*.app', System.IO.SearchOption.AllDirectories))" >
```

Comment:
I see net9.0. Can we make this a variable or at least some check via XMLPeek to use the actual version in our props. For example, we have this in a place:

```xml
    <XmlPeek
        Condition="Exists('$(MaciosSrcPath)\eng\Versions.props')"
        XmlInputPath="$(MaciosSrcPath)\eng\Versions.props"
        Query="/Project/PropertyGroup/MicrosoftNETCoreAppRefPackageVersion/text()">
      <Output TaskParameter="Result" PropertyName="VersionFromMacios" />
    </XmlPeek>
```

I know we got `_MauiDotNetVersionMajor` in Directory.Build.props

But maybe that file is already imported and we can just use the `_MauiDotNetTfm` or even the `MauiDeviceTestsPlatforms` values.